### ISSUE TITLE
Agarplus.io minimap works now.

### ISSUE THREAD

**Issue by KingMatt123** [davidmann4/ogar-feeder-bot#259]
positioning map now works with agarplus.io

**Comment by KingMatt123** [comment]
nvm u need to get to the corner

**Comment by hassanila97** [comment]
@AmazingGamerBoy Nope, for me works without corner

**Comment by davidmann4** [comment]
corners make it possible to calculate the offset

**Comment by firelightning13** [comment]
In gameserver.ini, change:

```
borderLeft = -7071
borderRight = 7071
borderTop = -7071
borderBottom = 7071
```

It will work for agarplus minimap

**Comment by SrNicolas** [comment]
actually doesnt need to go to the corner :p

**Comment by hassanila** [comment]
@SrNicolas Yes you need to go to the corner... Test it and keep your eyes on it, you will see that it's just an illusion until you reach a border then it gets calibrated

**Comment by firelightning13** [comment]
@SrNicolas It does work in dimotsai agar minimap but doesnt work in agarplus like hassanila97 mentioned

**Comment by davidmann4** [comment]
i guess srNicolas is right - on ogar servers you do not have to go to the corners
